cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java

```diff
@@ -80,42 +81,61 @@ public Integer call() throws Exception {
         @Option(names = { "-f", "--force" }, required = false, description = "Force deletion without confirmation.")
         Boolean force;
 
+        private Map<String, ServiceBean> readServicesFromYAMLFile() throws IOException {
+            SpecBean specBean = YamlMapper.loadBean(yamlCommandOptions.getYamlFilePath(), SpecBean.class);
+            return specBean.getServices();
+        }
+
         @Override
         public Integer call() throws Exception {
+            Map<String, ServiceBean> services = readServicesFromYAMLFile();
 
-            if (force != null) {
-                doRemoveServiceInstance(yamlCommandOptions);
-            } else {
+            // in case the --force flag is not specified, we shall prompt the user to prevent the removal of services
+            // as that might remove valuable data a swell
```

Comment:
Little typo:
```suggestion
            // as that might remove valuable data aswell
```

Reply:
Should be

```suggestion
            // as that might remove valuable data as well
```

though. Thanks for the tip, though!

Reply:
I doubt that: https://www.dict.cc/?s=aswell https://www.urbandictionary.com/define.php?term=aswell

Reply:
as well, or aswell :P

Reply:
Whatever dictionary (not the web stuff) I ask, "aswell" is simply wrong. Going for "as well".